### ISSUE TITLE
gap between groups of dodged boxplots should be greater than within

### ISSUE THREAD

**Issue by mhinsch** [GiovineItalia/Gadfly.jl#1285]

**Comment by Mattriks** [comment]
See my https://github.com/GiovineItalia/Gadfly.jl/issues/1092#issuecomment-449600709 for a potential workaround

**Comment by mhinsch** [comment]
> See my [#1092 (comment)](https://github.com/GiovineItalia/Gadfly.jl/issues/1092#issuecomment-449600709) for a potential workaround
> 
> Also the [dev docs example](http://gadflyjl.org/dev/gallery/geometries/#%5BGeom.boxplot%5D(@ref)-1) uses `Theme(boxplot_spacing= )`.

Yes, I have seen both. boxplot_spacing is what I am using for now (manuscript is due in two days...) to make the plot at least non-ugly, but the underlying issue remains - there is no difference in gap width between and within groups created by Stat.dodge.

Subplot_grid looks nice in terms of spacing, however a) the boxplots get cut off at the borders of the individual parts of the grid and b) I wasn't able to find out how to customize x axis ticks and labels in this case (although, TBH, I didn't spend too much time looking (for reasons mentioned above)).

**Comment by Mattriks** [comment]
When you have time, can you provide a minimal code example of your plot with subplot_grid.
